Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -108,6 +112,81 @@ - (void)requestOneTimePayment:(BTPayPalCheckoutRequest *)request
 
 #pragma mark - Helpers
 
+- (void)startPayPalNativeCheckout:(BTPayPalCheckoutRequest *)request
+                    configuration:(BTConfiguration *)configuration
+                        pairingId:(NSString *)payToken
+                       completion:(void (^)(BTPayPalAccountNonce *tokenizedCheckout, NSError *error))completionBlock {
```

Comment:
Nitpick - this can be simplified

```suggestion
                       completion:(void (^)(BTPayPalAccountNonce *, NSError *))completionBlock {
```